### DISCUSSION TITLE
how can I do to replace the "torch.distributed.barrier()"

### DISCUSSION THREAD

**Discussion by ybbthou8** [horovod/horovod#3093] [Q&A]
I want to modify code of mmdet to support horovod (just replace ddp func to horovod func), however, pytorch has "torch.distributed.barrier()", I donnot find homologous barrier func in horovod, how can I do to replace the "torch.distributed.barrier()"?

**Comment by maxhgerlach** [comment]
There is no `hvd.barrier()` yet in Horvod.

Are you working with MPI? Using mpi4py's barrier usually works fine for my purposes, as explained in this issue https://github.com/horovod/horovod/issues/1380 .
